src/i18n/messages/hi.json

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
Hello @igobranco!

You also need to import Hindi messages here https://github.com/openedx/frontend-app-authn/blob/4d1922c3b3617028a7832da66c82634427590f4f/src/i18n/index.jsx#L1-L33 and add the mapping into messages object.

Reply:
Ok.
I've included also the new languages that is the objective of this PR.